sound/soc/sof/ipc4-priv.h

```diff
@@ -40,6 +40,7 @@ struct sof_ipc4_fw_data {
 	void *nhlt;
 	enum sof_ipc4_mtrace_type mtrace_type;
 	u32 mtrace_log_bytes;
+	int num_ppl;
```

Comment:
also add to the description in the comment above? and maybe rename this to max_num_pipelines?